Application/EdFi.Ods.SandboxAdmin/Program.cs

```diff
@@ -21,15 +21,6 @@ public static async Task Main(string[] args)
             ConfigureLogging();
 
             var host = Host.CreateDefaultBuilder(args)
```

Comment:
confused here, in the test harness we have that logic to add the secrets, do we need to add that same logic here?

Reply:
this is covered by the default builder